### PR TITLE
Add aggregation of the enrollment and unenrollments.

### PR DESCRIPTION
**Pull request by flying-pi**
https://youtrack.raccoongang.com/agiles/104-65/105-71?issue=IA-88

### CODE REVIEW

File: rg_instructor_analytics/tasks.py [hunk 1]
```diff
@@ -85,8 +84,24 @@ def enrollment_collector_date():
         for enrol in EnrollmentByStudent.objects.all().values("course_id", "student", "last_update", "state", )
     }
 
-    total_stat = cache.get(ENROLLMENT_STAT_CACHE_BY_COURSE_KEY, {})
     result_stat = {}
+
+    exist_stat = EnrollmentTabCache.objects.filter(
+        created__exact=RawSQL(
```
Comment (idegtiarov) [inline rg_instructor_analytics/tasks.py:90]: Have you tried to use Django subquery for the same request `https://docs.djangoproject.com/en/2.0/ref/models/expressions/#subquery-expressions`?

**Review by idegtiarov — CHANGES_REQUESTED** [review]
One question is inline. Please take a look.

File: rg_instructor_analytics/tasks.py (hunk 1 above)
Comment (flying-pi) [reply rg_instructor_analytics/tasks.py]: Because 2 CourseKey objects for the same course has a different hash.

Comment (flying-pi) [reply rg_instructor_analytics/tasks.py:90]: We can`t use this feature, because this it available from Django 1.11, but OpenEdx support only 1.8.

Comment (idegtiarov) [reply rg_instructor_analytics/tasks.py:90]: Thanks for the explanation.

**Review by idegtiarov — APPROVED** [review]

**Pull request merged**